ceph/rados/core_workflows.py

```diff
@@ -4615,3 +4615,68 @@ def get_rados_df(self, pool_name: str = None):
         out = self.run_ceph_command(cmd=_cmd, client_exec=True)
 
         return out["pools"][0] if pool_name else out
+
+    def set_configuration(self, who: str, param: str, value: str) -> int:
```

Comment:
Module for this already exists. please refer to `set_config()` in `tests/rados/monitor_configurations.py`

Reply:
Reused existing function set_config()
` config_obj.set_config(
                section=config_section,
                name=config_name,
                value=config_value,
            )`
https://github.com/red-hat-storage/cephci/blob/262205193a7497c6e25014f51a479518e7a1ba85/tests/rados/test_bz_2213766.py#L69-L73